[terra-content-container] Adding backgroundColor prop in content-container (#3817)
cerner/terra-core: packages/terra-scroll/CHANGELOG.md

* Added
  * Added styles when the scroll is focussed

> * Changed
>   * Removed the focus style changes added for terra-scroll.

Reviewer: (update `a.b.c` to the appropriate version)
```suggestion
* Changed
  * Reverted the focus style changes added in `a.b.c`.
```

Author: Updated
